### ISSUE TITLE
updated ran it now wont open

### ISSUE THREAD

**Issue by Dictator5869** [builtbybel/xd-AntiSpy#66]
I updated from version 4.11 to the latest, ran the program worked fine, then went to open it again to check something now it wont open and even the older version wont open.

**Comment by Belim** [comment]
Seems like the problem isn't with the app, but with Windows 💭  Any error messages? Have you tried a fresh re-download?

**Comment by Dictator5869** [comment]
Sorry i was meant to update this with more info as I made it in a rush,

so I actually ended up doing a system restore after I posted this due to a issue with graphic drivers but it turns out version 4.11.100 still works but not any version after that, I have tried multiple different downloads and different extraction locations. when i try to open it by either double clicking or right click open even right click open as admin nothing happens, mouse loading spins for like a nano second then nothing.  I have just tried re downloading it no and still the same issue.
